### PR TITLE
Fixed a couple of bitesize bugs

### PR DESCRIPTION
**Pull request by krnekhelesh**
Fixes Issue #45, #108 and #109

### CODE REVIEW

File: plugins/github/IssuePage.qml [hunk 1]
```diff
@@ -299,26 +299,26 @@ Page {
             }
 
             //TextArea {
-//                id: textArea
```
Comment (krnekhelesh) [inline plugins/github/IssuePage.qml:302]: I pressed Ctrl+i to automatically indent the entire file. This is why you see this code diff. If you want I can undo this change.

**Comment by krnekhelesh** [comment]
I have only improved the label style in the IssuePage.qml. I haven't changed it in other places like the IssuesPage.qml. I wanted to confirm with you before I do that.

**Comment by iBelieve** [comment]
Thanks for your work on this!

Regarding the labels on the issues list page and in the labels editor, I'm thinking its probably best to keep just the labels names and color them on the list page so it doesn't get overly cluttered.

And on the labels editor, maybe go with the square UbuntuShapes to the left of the labels, since we can't really do a Flow layout with the selector checkboxes?

Do you want to do that in this pull request or should I go ahead and merge this?

**Comment by krnekhelesh** [comment]
Yeah I agree to the suggestion above. I will add the square UbuntuShapes to left of the labels (similar to Ubuntu Tasks) in this pull request since it should only take a few lines of code to implement.

**Comment by iBelieve** [comment]
Great, thanks!

**Comment by krnekhelesh** [comment]
Here is how it looks now https://imgur.com/kFvMJVG

**Comment by iBelieve** [comment]
Looks great!

**Comment by iBelieve** [comment]
Are you ready to have it merged?

**Comment by krnekhelesh** [comment]
Yup I think it is ready now. Good to merge.

**Pull request merged**